vpr/src/base/vpr_api.cpp

```diff
@@ -1183,6 +1188,29 @@ bool vpr_analysis_flow(t_vpr_setup& vpr_setup, const t_arch& Arch, const RouteSt
         VTR_LOG("*****************************************************************************************\n");
     }
 
+    /* If routing is successful, apply post-routing annotations
+     * - apply logic block pin fix-up
+     *
+     * Note: 
+     *   - Turn on verbose output switch when you want to debug
```

Comment:
Shouldn't this be controlled by command line flags?  We have `--pack_verbosity`?